app/graphql/types/user_type.rb

```diff
@@ -5,5 +5,10 @@ class UserType < Types::BaseObject
     field :username, String, null: false
     field :enneagram, Types::EnneagramType, null: true
     field :myers_brigg, Types::MyersBriggType, null: true
+    field :friends, [Types::UserType], null: true
```

Comment:
interesting - it looks like if you want an array of data you put `Types::UserType` in brackets.. cool!